INSTALL.md

```diff
@@ -0,0 +1,89 @@
+This file gives a detailed explanation of how to install JobAdder on user clients, the server, or worker clients.
+Note that all components of JobAdder were designed to be used exclusively on Linux.
+Furthermore, the server and the worker clients are assumed to be using central authentication (e.g. LDAP).
+
+## Requirements
+The following software is required to run JobAdder:
+
+- Python 3.7 or higher (user client, server, and worker clients)
+- PostgreSQL (server)
+- Docker (worker clients)
```

Comment:
We need a version, at least say "recent version" (some distros do have very outdated docker)